### PR TITLE
Insurance inactivate.

### PR DESCRIPTION
**Pull request by teryhill**
Deactivate an insurance company #848

### CODE REVIEW

**Comment by Trodrige** [comment]
@teryhill I don't observe any change after deactivating and saving. Check the deactivate box, save and nothing has updated. I observe same for address and phone fields.

**Comment by teryhill** [comment]
@aethelwulffe  did you test?

**Comment by teryhill** [comment]
PR sent to add needed files and inc version

**Comment by tmccormi** [comment]
Need @aethelwulffe  now :-)

**Comment by aethelwulffe** [comment]
Starting my reply while pulling down branch....
OK... Installing test DB....
Darn, applying DB changes by hand....my script dropped the table...

File: library/classes/InsuranceCompany.class.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 /************************************************************************
-  			InsuranceCompany.php - Copyright duhlman
```
Comment (aethelwulffe) [inline library/classes/InsuranceCompany.class.php]: This attribution OK in this project?

**Comment by aethelwulffe** [comment]
OK, deactivated, and it is not showing anymore.  Good feature.  Not sure what happens to patients that have that insco, just that it can't be selected anymore.  Probably need a "merge" tool too.

**Pull request merged**